newm-server/src/test/kotlin/io/newm/server/features/user/UserRoutesTests.kt

```diff
@@ -29,15 +33,38 @@ import io.newm.server.features.user.model.UserIdBody
 import io.newm.server.model.ClientPlatform
 import io.newm.shared.koin.inject
 import io.newm.shared.ktx.existsHavingId
+import io.newm.shared.ktx.getConfigString
 import io.newm.shared.ktx.toHash
 import kotlinx.coroutines.runBlocking
 import org.jetbrains.exposed.sql.deleteAll
 import org.jetbrains.exposed.sql.transactions.transaction
+import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
+import org.koin.core.context.GlobalContext.loadKoinModules
+import org.koin.dsl.module
 import java.time.LocalDateTime
 
 class UserRoutesTests : BaseApplicationTests() {
+    @BeforeAll
+    fun beforeAllSongRoutesTests() {
+        val module =
+            module {
```

Comment:
I already fixed the annoying ktlint formatting rule to that requireda  new line after each `=`.  You should be able to do `val module = module {`